deploy/manifests/nginx-gateway.yaml

```
@@ -122,7 +146,7 @@ spec:
         - static-mode
         - --gateway-ctlr-name=k8s-gateway.nginx.org/nginx-gateway-controller
         - --gatewayclass=nginx
-      - image: nginx:1.23
+      - image: nginx:1.24
```

Latest NGINX version is 1.25, and the docs would need to be updated, but Luca has a PR open for bumping this anyway, so this change should probably be left for that PR.
```suggestion
      - image: nginx:1.23
```